MVPTask/Sources/Scenes/RecipeDetailViewController/RecipeDetailViewController.swift

```diff
@@ -3,6 +3,14 @@
 
 import UIKit
 
+/// Состояния загрузки
+enum Loading {
```

Comment:
```suggestion
enum Loading {
```
на будущее см важное там есть StateView пригодится с сетью